src/test/resources/fake-agency-bad-calendar-date/routes.txt

```diff
@@ -1,2 +1,7 @@
 agency_id,route_id,route_short_name,route_long_name,route_desc,route_type,route_url,route_color,route_text_color,route_branding_url
 1,1,1,Route 1,,3,,7CE6E7,FFFFFF,
+
+
+
+
```

Comment:
Are these extra lines intentional?

Reply:
Yea, the test fails on a file like this with the current code.